lib/raw_file.dart

```diff
@@ -0,0 +1,109 @@
+// Copyright 2022 The wav authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:typed_data';
+
+import 'wav_bytes_reader.dart';
+import 'wav_bytes_writer.dart';
+import 'wav_no_io.dart' if (dart.library.io) 'wav_io.dart';
+import 'wav_types.dart';
+
+class Raw {
```

Comment:
It looks like this class is all static methods. Dart style [discourages this pattern](https://dart.dev/effective-dart/design#avoid-defining-a-class-that-contains-only-static-members). Just make them top level functions instead: `readRawAudioFile`, `readRawAudio`, etc.